core/src/main/java/it/gov/pagopa/rtd/transaction_filter/service/store/AggregationData.java

```diff
@@ -0,0 +1,25 @@
+package it.gov.pagopa.rtd.transaction_filter.service.store;
+
+import lombok.Data;
+import lombok.ToString;
+
+@Data
+@ToString
+public class AggregationData {
+
+  private short numTrx;
```

Comment:
A short can be a little strict in future scenarios (virtual POS, aggregators..).. upgrade to int